symengine/printers/sbml.cpp

```diff
@@ -0,0 +1,154 @@
+#include <symengine/printers/sbml.h>
+#include <symengine/printers.h>
+
+namespace SymEngine
+{
+
+static std::vector<std::string> init_sbml_printer_names()
+{
+    std::vector<std::string> names = init_str_printer_names();
+    names[SYMENGINE_LOG] = "ln";
+    names[SYMENGINE_ASIN] = "arcsin";
+    names[SYMENGINE_ACOS] = "arccos";
+    names[SYMENGINE_ASEC] = "arcsec";
+    names[SYMENGINE_ACSC] = "arccsc";
+    names[SYMENGINE_ATAN] = "arctan";
+    names[SYMENGINE_ACOT] = "arccot";
+    names[SYMENGINE_ASINH] = "arcsinh";
```

Comment:
First I thought that it should be `arsinh`, but then I saw that it is actually called `arcsinh` or `asinh` in SBML. Would it be better to use the `asinh` alternative here? In the SBML documentation these forms are listed first.

Reply:
libsbml & jsbml (the two main libraries for working with sbml) both use the long form, I guess to be consistent with mathml:

https://github.com/sbmlteam/mathml-tests/blob/master/tests/mathML_test_0038.xml

Reply:
Ok, in that case you should keep it.